info/available-rules.md

```diff
@@ -1,131 +1,131 @@
-| 2    | 2.1.1    | MISSING_KDOC_CLASS_ELEMENTS               | Check: warns if accessible internal elements (protected, public, internal) in a class are not documented<br>Fix: no                                                                                                                                          | no  | no                                                                                                                                                                                                                         | May be need to add exception cases for setters and getters. There is no sense in adding KDoc to them.                                                                                                                         |
+| 1    | 1.6.1    | VARIABLE_NAME_INCORRECT_FORMAT            | Check: warns if the name of the variable is not in lowerCamelCase or contains non-ASCII letters<br>Fix: fixing the case format to lowerCamelCase                                                                                                                 |     | no                                                                                                                                                                                                                         | -                                                                                                                                                                                                                             |
+| 1    | 1.6.2    | FUNCTION_BOOLEAN_PREFIX                   | Check: functions/methods that return boolean should have a special prefix, such as "is/should/e.t.c"<br>Fix:                                                                                                                                                       | yes | no                                                                                                                                                                                                                         | Recursively update usages of this function in the projectAggressive fix - what if the new name will not be valid?                                                                                                                 |
```

Comment:
The following sentence is not clear to me:
"Recursively update usages of this function in the projectAggressive fix - what if the new name will not be valid?"
Please, explain.

Reply:
> Recursively updates usages of this function in the project. Aggressive fix &mdash; what if the new function name will still be invalid?

Reply:
OK.